Calibration/EcalCalibAlgos/interface/ECALpedestalPCLworker.h

```diff
@@ -53,9 +53,16 @@ class ECALpedestalPCLworker : public  DQMEDAnalyzer {
     std::vector<MonitorElement *> meEB_;
     std::vector<MonitorElement *> meEE_;
 
-    static const int kPedestalSamples = 3;
-    static const int kThreshold = 10; // threshold (in adc count) above which we'll assume 
-                                      // there's signal and not just pedestal
+    uint32_t pedestalSamples_ ; // number of presamples to be used for pedestal determination    
+    bool     checkSignal_;      // avoid frames containing a signal
+    uint32_t  sThreshold_ ;     // if checkSignal = true threshold (in adc count) above which we'll assume 
+                                // there's signal and not just pedestal
```

Comment:
hello @argiro 

With the noise being significantly different between EB and EE (factor 2-3),
and somewhat |eta| dependent in EE
the significance to have "signal" in EB and EE dataframes
could be set here in relative terms in units of of sigma's of the noise
and the check carried out against Numsigna's x sigma_in_db